src/dpo/test/aes.py

```diff
@@ -0,0 +1,15 @@
+from openroad import Design, Tech
+import helpers
+
+tech = Tech()
+tech.readLef("Nangate45/Nangate45.lef")
+
+design = Design(tech)
+design.readDef("aes.def")
+
+design.getOptdp().improvePlacement(1, 0, 0)
+design.evalTclString("check_placement")
```

Comment:
Couldn't you call this through dpl?

Reply:
Good catch. I think I wrote these before dpl was wrapped. There are others that can probably be removed as we finish the Python wrapping. Will need to make another pass when most of the wrapping is done, but I have fixed these.